released/packages/coq-pil/coq-pil.1.0.1/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Coq library for Propositional Intuitionistic Logic & Pitts Interpolation Library"
+
+homepage: "https://github.com/hferee/rocq-pil"
+dev-repo: "git+https://github.com/hferee/rocq-pil.git"
+bug-reports: "https://github.com/hferee/rocq-pil/issues"
+doc: "https://hferee.github.io/UIML"
+maintainer: "feree@irif.fr"
+authors: [
+  "Hugo Férée"
+  "Sam van Gool"
+  "Yago Iglesias Vasquez"
+]
+license: "CECILL-2.1"
+
+depends: [
+  "dune" {>= "3.8"}
```

Comment:
```suggestion
```
Your project appears to build using `coq_makefile`, so how does it depend on Dune? From what I can tell, this is not a plugin.